src/components/banners/alert/alert.component.html

```diff
@@ -1,8 +1,28 @@
-<div class="alert alert-{{type}}" role="alert">
-  <div class="alert-container">
-    <i class="icon {{type | cutAlertIconClass}}" role="presentation" [attr.aria-hidden]="!showIcon" *ngIf="showIcon"></i>
-    <div class="alert-message" data-ng-transclude>
+<div [ngClass]="{'hmcts-banner hmcts-banner--warning':type === alertMessageType.WARNING || type === alertMessageType.ERROR}">
+  <div [ngClass]="{'hmcts-banner hmcts-banner--success':type === 'success'}">
```

Comment:
Why do we have have an outer div with a warning class and an inner div with a success class?

Reply:
Also, why are WARNING and ERROR using constants but success is just a string?

Reply:
To answer the first question, I couldn't get all the styles set on one div. I think because the subscription of the observers goes through this code three times the class would be the same for all of them if we wanted to put it in one div (even with ngClass). I attempted to get the styling correct for all messages with more clarity but this was the simplest way I could find that displayed the desired styling.

Reply:
Don't know why I've left success as a string though, I'll change that

Reply:
What are you thoughts on my answer with regard to the first question? I could replicate the issue I was getting with ngClass and demo it to you to indicate the issues I was having with regards to that styling?